ArrtModel/Model/Settings/ArrAccountSettings.cpp

```diff
@@ -21,6 +21,19 @@ ArrAccountSettings::ArrAccountSettings(QObject* parent)
         {"UK South", "uksouth.mixedreality.azure.com"},
         {"West Europe", "westeurope.mixedreality.azure.com"},
         {"West US", "westus2.mixedreality.azure.com"}};
+
+    // supported account domains
+    m_supportedAccountDomains = {
+        {"Australia East", "australiaeast.mixedreality.azure.com"},
+        {"East US", "eastus.mixedreality.azure.com"},
+        {"East US 2", "mixedreality.azure.com"},
```

Comment:
just making sure this is correct (mixedreality.azure.com with no prefix)

Reply:
this is correct and will correspond to "sts.mixedreality.azure.com"